### PR TITLE
create hooks and tests scenarios for account contact and opportunity

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/test/java/salesforce/hooks/AccountHooks.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+package salesforce.hooks;
+
+import api.ApiManager;
+import api.ApiMethod;
+import api.ApiRequest;
+import api.ApiResponse;
+import api.ApiRequestBuilder;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
```
Comment (GustavoZHA) [inline src/test/java/salesforce/hooks/AccountHooks.java]: Delete extra line

File: src/test/java/salesforce/steps/ContactGeneralSteps.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    @Then("the response status code should be {string} to contact")
+    public void theResponseStatusCodeShouldBeToContact(String statusCode) {
+        log.info("I verify status Contact response");
+        Assert.assertEquals(apiResponse.getStatusCode(), HttpStatus.valueOf(statusCode).value());
+        apiResponse.getResponse().then().log().body();
+    }
+
+
```
Comment (GustavoZHA) [inline src/test/java/salesforce/steps/ContactGeneralSteps.java]: Delete blank line

File: src/test/java/salesforce/steps/OpportunityGeneralSteps.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        apiResponse = ApiManager.execute(apiRequest);
+    }
+
+    @Then("the response status code should be {string} to opportunity")
+    public void theResponseStatusCodeShouldBeToOpportunity(String statusCode) {
+        log.info("I verify status Opportunity response");
+        Assert.assertEquals(apiResponse.getStatusCode(), HttpStatus.valueOf(statusCode).value());
+        apiResponse.getResponse().then().log().body();
+    }
+
```
Comment (GustavoZHA) [inline src/test/java/salesforce/steps/OpportunityGeneralSteps.java]: here too

File: src/test/java/salesforce/hooks/AccountHooks.java (hunk 1 above)
Comment (jpgonzales27) [reply src/test/java/salesforce/hooks/AccountHooks.java]: ready gus, I already corrected everything

File: src/test/java/salesforce/steps/OpportunityGeneralSteps.java (hunk 3 above)
Comment (jpgonzales27) [reply src/test/java/salesforce/steps/OpportunityGeneralSteps.java]: ready gus, I already corrected everything

**Pull request merged**